src/TUTORIAL.md

```diff
@@ -1016,7 +1016,26 @@ This is similar to what we've already done in the `accounts.js` handler, but mor
 
 ### Preventing Referred Records Deletion
 
-Another situation we must gracefully prevent is deleting _Product_ and _Account_ records when _Order_ records exist for them. Let's add the appropriate hooks to our `account.js` handler:
+Another situation we must gracefully prevent is deleting _Product_ and _Account_ records when _Order_ records exist for them. 
+
+Currently, (assuming a product with ID 1 has been purchased by a customer with an account which also has an ID of 1)... 
+
+...if you try to delete a product with ID 1:
+
+```shell
+$ curl -v -X DELETE http://localhost:3001/products/1
+```
+...or delete an account with ID 1
+
+```shell
+$ curl -v -X DELETE http://localhost:3001/accounts/1
+```
+
+...the web-service will return another [HTTP 500](https://tools.ietf.org/html/rfc7231#section-6.6.1) error, with an unhelpful error message: "Internal server error".  
+
+
+
```

Comment:
Let's have a single blank line to separate paragraphs consistent with the rest of the document.